dashboard/src/pages/TreeDetails/Tabs/Boots/BootsTable.tsx

```diff
@@ -0,0 +1,312 @@
+    header: ({ column }): JSX.Element =>
+      TableHeader({
+        column: column,
+        sortable: true,
+        intlKey: 'testDetails.status',
+        intlDefaultMessage: 'Status',
+        tooltipId: 'bootsTab.statusTooltip',
+      }),
+  },
+  {
+    accessorKey: 'startTime',
+    header: ({ column }): JSX.Element =>
+      TableHeader({
+        column: column,
+        sortable: true,
+        intlKey: 'buildDetails.startTime',
+        intlDefaultMessage: 'Start time',
+      }),
+    cell: ({ row }): JSX.Element => (
+      <TooltipDateTime
+        dateTime={row.getValue('startTime')}
+        lineBreak={true}
+        showLabelTime={true}
+        showLabelTZ={true}
+      />
+    ),
+  },
+  {
+    accessorKey: 'duration',
+    header: ({ column }): JSX.Element =>
+      TableHeader({
+        column: column,
+        sortable: true,
+        intlKey: 'testDetails.duration',
+        intlDefaultMessage: 'Duration',
+      }),
+    cell: ({ row }): string =>
+      row.getValue('duration') ? row.getValue('duration') : '-',
+  },
+  {
+    id: 'chevron',
+    cell: (): JSX.Element => <MdChevronRight />,
+  },
+];
+
+interface IBootsTable {
+  treeId: string;
+  testHistory: TestHistory[];
+}
+
+export function BootsTable({ treeId, testHistory }: IBootsTable): JSX.Element {
+  const [sorting, setSorting] = useState<SortingState>([]);
+  const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>([]);
+  const [pagination, setPagination] = useState<PaginationState>({
+    pageIndex: 0,
+    pageSize: 10,
+  });
+  const { tableFilter } = useSearch({
+    from: '/tree/$treeId/',
+  });
+  const [bootsSelectedFilter, setBootsSelectedFilter] =
+    useState<TestsTableFilter>(tableFilter.bootsTable);
+
+  const navigate = useNavigate({ from: '/tree/$treeId' });
+  const intl = useIntl();
+
+  const rawData = useMemo(
+    (): TTestByCommitHashResponse => ({
+      tests: testHistory.map(
+        (e): TestByCommitHash => ({
+          duration: e.duration?.toString() ?? '',
+          id: e.id,
+          path: e.path,
+          startTime: e.startTime,
+          status: e.status,
+        }),
+      ),
+    }),
+    [testHistory],
+  );
+
+  const data = useMemo((): TestByCommitHash[] => {
+    const filterToApply = tableFilter.bootsTable;
+    if (filterToApply === 'all') {
+      return rawData?.tests;
+    }
+    return rawData?.tests.filter(test => {
+      return getStatusGroup(test.status) === filterToApply;
+    });
+  }, [rawData?.tests, tableFilter]);
+
+  const filterCount: Record<(typeof possibleTestsTableFilter)[number], number> =
+    useMemo(() => {
+      const count = {
+        all: 0,
+        success: 0,
+        failed: 0,
+        inconclusive: 0,
+      };
+      count.all = rawData.tests.length;
+      rawData.tests.forEach(test => count[getStatusGroup(test.status)]++);
+
+      return count;
+    }, [rawData.tests]);
+
+  const onClickRow = useCallback(
+    (id: string) => {
```

Comment:
Use TableCellWithLink for navigation, programtical navigation will make us lose the right click functionalities of a link

Reply:
Unfortunately there is no correct way to have a single link for the entire table row